wakepy/methods/wsl.py

```diff
@@ -0,0 +1,100 @@
+import enum
+from abc import ABC, abstractmethod
+from subprocess import Popen, PIPE
+from typing import List
+
+from wakepy.core import DbusAdapter, Method, ModeName, PlatformName
+
+class Flags(enum.StrEnum):
```

Comment:
The  [enum.StrEnum](https://docs.python.org/3/library/enum.html#enum.StrEnum) is unfortunately available only for python 3.11 and above. I've created my own backport which can be imported from wakepy.core. Let's use that instead so we can keep wakepy supported from python 3.7 onwards.